packages/client/components/SwipeableDashSidebar.tsx

```diff
@@ -111,6 +111,7 @@ const SwipeableDashSidebar = (props: Props) => {
     openPortal()
     return () => {
       window.clearTimeout(swipe.peekTimeout)
+      swipe.showBodyScroll && swipe.showBodyScroll()
```

Comment:
Interesting... I'll give it a change. If it doesn't cause a regression (which I doubt it would), seems "safe to try"